Improve documentation

rm-- opened the pull request:
fixes #1

keimlink commented on docs/index.rst:
```diff
@@ -5,11 +5,21 @@ Durga: Easy Python REST resources
+--------
+* Lightweight REST client
+* Easy to use concept consisting of Resource, Collection, Element
+* Describe a ressource as schema and validate incoming JSON against it
```
Typo "ressource"

keimlink commented on docs/installation.rst:
```diff
@@ -10,3 +12,10 @@ Or, if you have virtualenvwrapper installed::
+===========
+You can install `durga` from source just as you would install any other Python package::
+
+    $ pip install git+https://github.com/transcode-de/durga.git
```
Installing this way is not a good idea:
1. You have to have git installed.
2. This will clone the repository which takes some time but has no advantage.
3. You get the latest revision from the main branch.

The question is: Why should someone install the distribution not from PyPI?

keimlink commented:
Thanks for the pull request! There seems to be a problem with the fixture for the Discogs test/example. MusicBrainz has a [REST API](http://wiki.musicbrainz.org/Development/JSON_Web_Service) which may be a better choice. It has RESTful URLs and the data is Open Data. Please see if you can replace Discogs with MusicBrainz.

rm-- commented:
Example replaced with MusicBrainz.

rm-- committed:
Added new example and fixed file permissions

MusicBrainzResource got no objects_path and so in  current implementation the
test fails. Thats the reason for xfail marker at test.

rm-- committed:
Improved documentation and changed Sphinx theme

rm-- committed:
Made README PYPI-readable

keimlink commented on docs/conf.py:
```diff
@@ -55,7 +54,7 @@
 # General information about the project.
 project = u'Durga'
-copyright = u'2014, transcode'
+copyright = u'2014-{}, {}'.format(date.today().year, 'transcode')
```
Author name is a string and needs not to be inserted using `format()`.

keimlink commented on docs/index.rst:
```diff
@@ -5,11 +5,24 @@ Durga: Easy Python REST resources
 
+Features
+--------
+* Lightweight REST client
```
Typo: "Light weight"

keimlink commented on docs/index.rst:
```diff
@@ -5,11 +5,24 @@ Durga: Easy Python REST resources
+* Easy to use API concept consisting of Resource, Collection, Element
+  corresponds to `Django`_ ORM with Model, Manager, Instance
+* Describe a resource as schema and validate incoming JSON against it
+* Compatible with Python 2.x >= 2.7 or 3.x >= 3.4
```
Please write in a human readable way: "Python 2.7 and 3.4"

rm-- committed:
Corrected isort errors

keimlink commented:
Thanks René!
